### DISCUSSION TITLE
[Feature] Add spy permission

### DISCUSSION THREAD

**Discussion by alowave223** [Zrips/CMI#4874] [Feature Requests]
Greetings! Recently faced a problem, specifically with the chat and the fact that the administrators cannot spy the local chat correctly, the bypassRange permission does not work as expected. When i give this permission, the administrator is forced to write public messages.
Suggestion: Add permission that would relay local chat for the administration without being forced to write public messages like bypassRange does. An example can be seen in essentials there is a similar ability to give the essentials.chat.spy permission.

**Comment by alowave223** [comment]
up

**Comment by bigenergy** [comment]
+1

**Comment by VitSun2001** [comment]
+1
That feature should have been added ages ago

**Comment by div2005** [comment]
+1

**Comment by alowave223** [comment]
Why is this request being ignored? Why can't you do what the people who bought this plugin really need? It's not difficult

**Comment by bigenergy** [comment]
@Zrips pls add

**Comment by bigenergy** [comment]
Since the author is simply ignoring this suggestion from several people, I urge you to write a review on the page https://www.spigotmc.org/resources/cmi-280-commands-insane-kits-portals-essentials-economy-mysql-sqlite-much-more.3742/reviews

Perhaps the author will come to his senses and will take into account the opinion of several buyers, including the same offer hanging from May 2020... https://github.com/Zrips/CMI/discussions?discussions_q=spy

**Reply by mercurialmusic** [reply]
Comments like this are the reason devs abandon plugins. You bought the plugin as is. Zrips owes you nothing. It's fine to make feature requests, but that's up to the dev if they want to implement it in their plugin. 

Please chill with the entitlement.

**Reply by alowave223** [reply]
The reason of comments like this is a complete irresponsibility and tactlessness of the developer who ignores the requests of many plugin users for almost six months, all the more, it will not be difficult for him to do it, believe me, two lines of code will not take so much time to him =)

About what you say 'if they want to implement it in their plugin', this plugin is considered as public, that is, for everyone, which means if the developer does not take into account the opinion of the majority, he simply does not care

**Reply by bigenergy** [reply]
> Comments like this are the reason devs abandon plugins. You bought the plugin as is. Zrips owes you nothing. It's fine to make feature requests, but that's up to the dev if they want to implement it in their plugin.
> 
> Please chill with the entitlement.

Tell it to others, I'm not the only one asking to add this option, there is still a topic from the summer of 2020 https://github.com/Zrips/CMI/discussions/3510
ok?

**Reply by rbluer** [reply]
"this plugin is considered as public"  Seriously?  You gotta be kidding? This plugin is far from being considered in the public domain.

I won't tell you to cut him some slack, but you really should.  I'm sure he has valid reasons for not being able to address every request that is made, no matter how important it may be to yourself.  But encouraging others to malign and besmirch his name and the plugin will only result in your request being delayed even farther; if it ever will be addressed. 

Its probably not too late to change your approach and to find a more encouraging way to start discussions to find a resolution to this issue, and I hope you're able to.  Best wishes to finding an amiable solution.

**Reply by VitSun2001** [reply]
Btw request of this feature were coming since 2018 https://github.com/Zrips/CMI/issues/853
